pkg/internal/htmx/htmx.go

```
@@ -112,12 +146,43 @@
+	if len(identityList) == 0 {
+		value, err := renderTemplate(EmptyListTemplate, []string{})
+		if err != nil {
+			return fmt.Errorf("%w: %v", ErrExecutingTemplate, err)
+		}
+
+		w.Write([]byte(value))
+		return nil
+	}
+
+	data := []models.Identity{}
 	for _, identity := range identityList {
```

golangci-lint

S1011: should replace loop with `data = append(data, identityList...)`

(gosimple)